tests/e2e/login/privacy-policy.wdio-spec.js

```diff
@@ -0,0 +1,128 @@
+const commonElements = require('../../page-objects/common/common.wdio.page.js');
+const utils = require('../../utils');
+const loginPage = require('../../page-objects/login/login.wdio.page');
+const privacyPolicyPage = require('../../page-objects/privacy-policy/privacy-policy.wdio.page');
+const userFactory = require('../../factories/cht/users/users');
+const privacyPolicyFactory = require('../../factories/cht/settings/privacy-policy');
+const placeFactory = require('../../factories/cht/contacts/place');
+
+describe('Privacy policy', () => {
+  const englishTexts = privacyPolicyFactory.english;
+  const frenchTexts = privacyPolicyFactory.french;
+  const offlineUser = userFactory.build();
+  const onlineUser = userFactory.build({
+    username: 'online',
+    roles: ['program_officer'],
+    place: {
+      _id: 'hc2',
+      type: 'health_center',
+      name: 'Health Center 2',
+      parent: 'dist1'
+    },
+    contact: {
+      _id: 'fixture:user:onlineuser',
+      name: 'onlineuser'
+    }
+  });
+
+  const parent = placeFactory.place().build({_id:'dist1',type:'district_hospital'});
+
+  before(async () => {
+    await utils.saveDocs([parent, privacyPolicyFactory.privacyPolicy().build()]);
+    await utils.createUsers([ onlineUser, offlineUser]);
+  });
+
+  describe('for an online user', () => {
+    it('should show the correct privacy policy on login', async () => {
+      // After first login, check that privacy policy was prompted to user
+      await loginPage.login(onlineUser.username, onlineUser.password);
+      await browser.waitUntil(async () => {
+        const wrapperText = await (await privacyPolicyPage.privacyWrapper()).getText();
+        return wrapperText.includes(englishTexts.header) && wrapperText.includes(englishTexts.paragraph);
+      }, 'Timed out waiting for English Online Privacy Policy to Display');
+
+      // After accepting, no privacy policy on next load
+      await privacyPolicyPage.acceptPrivacyPolicy();
+      await browser.url('/');
+      expect(await privacyPolicyPage.privacyWrapper()).not.toBeDisplayed();
+
+      // Check display when loading privacy policy page
+      await privacyPolicyPage.goToPrivacyPolicyConfig();
+      await browser.waitUntil(async () => {
+        const wrapperText = await (await privacyPolicyPage.privacyConfig()).getText();
+        return wrapperText.includes(englishTexts.header) && wrapperText.includes(englishTexts.paragraph);
+      }, 'Timed out waiting for english online privacy to display');
+
```

Comment:
Can we please add another assertion here like:
```
      await browser.refresh();
      expect(await privacyPolicyPage.privacyWrapper()).not.toBeDisplayed();
```

So we test that the policy is not shown on refresh? 
Also, I think that every time we assert that the privacy is not shown, we should also add an assertion that something else we expect IS shown (like some header element or something).